assets/src/application/ui/display/DebugInfo.tsx

```diff
@@ -0,0 +1,38 @@
+import React, { CSSProperties } from 'react';
+import { Collapse } from 'antd';
+
+const { Panel } = Collapse;
+
+const dataStyle: CSSProperties = {
```

Comment:
can this be moved to a `.scss` file?

Reply:
Yes, but I thought it's a dev only component, so no need to extract to avoid bundling that with production CSS

Reply:
understood
`emotion` would be a great alternative here (just thinking out loud)